webui/react/src/components/ResourcePoolCard.module.scss

```diff
@@ -48,11 +48,11 @@ $line-height: 2.2rem;
         margin-bottom: 0.5rem;
       }
     }
-    .tags *[class^="Badge"] {
+    .tags *[class^='Badge'] {
```

Comment:
**nit:** You can actually just leave out the quotes here. Also it might be better to use `*=` instead of `^=` in case there are other classes added and order just happens to NOT have `Badge` come first. I think it's possible but not 100% sure.